parsers/US_SPP.py

```diff
@@ -0,0 +1,198 @@
+#!usr/bin/env python3
+
+"""Parser for the Southwest Power Pool area of the United States."""
+
+from dateutil import parser, tz
+from io import StringIO
+from logging import getLogger
+import pandas as pd
+import requests
+
+generation_url = 'https://marketplace.spp.org/public-data-api/gen-mix/asFile'
+
+exchange_url = 'https://marketplace.spp.org/public-data-api/interchange-trend/asFile'
+
+mapping = {'coal': 'coal',
+           'unknown': 'unknown',
+           'Wind': 'wind',
```

Comment:
Just checking - I can't load the source files, but do they really have "coal" lowercase and most everything else uppercase?

Reply:
Nope coal and the unknown key are leftovers that can be removed.